lib/spotlight/dor/indexer.rb

```diff
@@ -61,21 +80,36 @@ def add_donor_tags sdb, solr_doc
       insert_field solr_doc, 'donor_tags', donor_tags, :symbol # this is a _ssim field
     end
 
+    # add the folder number to solr_doc as folders_ssim
+    # TODO:  push this up to stanford-mods gem?  or should it be hierarchical series/box/folder?
+    def add_folder(sdb, solr_doc)
```

Comment:
`#add_box` and `#add_folder` look pretty similar. Would it make sense to pull some of this logic up to a method shared between them?